### ISSUE TITLE
What is the final audio loss Moshi achieved during pretraining?

### ISSUE THREAD

**Issue by ShinnosukeUesaka** [kyutai-labs/moshi#232]
### Due diligence

- [x] I have done my due diligence in trying to find the answer myself.

### Topic

The paper

### Question

I am training a model with audio tokens from Mimi.
